test/cli-commands/ganache/ganache.js

```diff
@@ -0,0 +1,130 @@
+
+	beforeEach(() => {
+		ganacheCommandOutput = '';
+	});
+
+	describe('Run ganache server on specific port', async () => {
+		it('should start ganache server on specific port', async () => {
+			runCmdHandler('./cli-commands/ganache', 'etherlime ganache --port 8123');
+			await timeout(RUNNING_GANACHE_TIMEOUT);
+			ganacheCommandOutput = getGanacheProcessState()['ganacheCommandOutput'];
+			const ganachePort = ganacheCommandOutput.substr(ganacheCommandOutput.length - 5).trim();
+
+			assert.equal(ganachePort, SPECIFIC_PORT, 'The ganache server is not running on specific port');
+		});
+	});
+
+	describe('Run ganache server and check accounts', async () => {
+		it('should start ganache server and validate accounts', async () => {
+			runCmdHandler('./cli-commands/ganache', 'etherlime ganache --port 8124');
+			await timeout(RUNNING_GANACHE_TIMEOUT);
+
+			ganacheCommandOutput = getGanacheProcessState()['ganacheCommandOutput'];
+
+			const rawAccountsString = ganacheCommandOutput.split(/\r?\n/).slice(0, 10);
+
+			const firstOutputtedAddress = rawAccountsString[0].substr(ADDRESS_START_INDEX, ADDRESS_LENGTH);
+			const firstOutputtedPrivateKey = rawAccountsString[0].substr(PRIVATE_KEY_START_INDEX, PRIVATE_KEY_LENGTH);
+
+			assert.equal(firstOutputtedAddress, FIRST_ACCOUNT_ADDRESS, 'There is mismatch of first account address');
+			assert.equal(firstOutputtedPrivateKey, FIRST_PRIVATE_KEY, 'There is mismatch of first account private key');
+
+			const thirdOutputtedAddress = rawAccountsString[2].substr(ADDRESS_START_INDEX, ADDRESS_LENGTH);
+			const thirdOutputtedPrivateKey = rawAccountsString[2].substr(PRIVATE_KEY_START_INDEX, PRIVATE_KEY_LENGTH);
+
+			assert.equal(thirdOutputtedAddress, THIRD_ACCOUNT_ADDRESS, 'There is mismatch of third account address');
+			assert.equal(thirdOutputtedPrivateKey, THIRD_PRIVATE_KEY, 'There is mismatch of third account private key');
+
+			const tenthOutputtedAddress = rawAccountsString[9].substr(ADDRESS_START_INDEX, ADDRESS_LENGTH);
+			const tenthOutputtedPrivateKey = rawAccountsString[9].substr(PRIVATE_KEY_START_INDEX, PRIVATE_KEY_LENGTH);
+
+			assert.equal(tenthOutputtedAddress, TENTH_ACCOUNT_ADDRESS, 'There is mismatch of tenth account address');
+			assert.equal(tenthOutputtedPrivateKey, TENTH_PRIVATE_KEY, 'There is mismatch of tenth account private key');
+		});
+	});
+
+	describe('Run ganache server on already used port', async () => {
+		it('should throw if we are trying to start ganache server on used port', async () => {
+			runCmdHandler('./cli-commands/ganache', 'etherlime ganache --port 8125', false);
+
+			await timeout(SECOND_TIMEOUT);
+
+			const stillRunningPID = getGanacheProcessState()['runningChildProcessPID'];
+
+			runCmdHandler('./cli-commands/ganache', 'etherlime ganache --port 8125');
+
+			await timeout(RUNNING_GANACHE_TIMEOUT);
+
+			killProcessByPID(stillRunningPID);
+
+			assert.isTrue(getGanacheProcessState()['portErrorOnProcess'], 'The ganache server is running on used port');
```

Comment:
It's a bit unreadable written like that. Try to get the result of get state in a variable and then probably use . notation or a constant with `[]` notation